### ISSUE TITLE
Update Albacete region record

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#2188]
Porting over from https://github.com/whosonfirst-data/whosonfirst-data-admin-es/issues/40

>I am opening this issue to address a discrepancy found in a pull request I submitted via the web form. https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/39
>
>The pull request involves changing the region from Albacete to Álava. This change isn't merely a translation alteration; rather, it corrects the geographical representation. Albacete is situated in the southern region of Spain, as depicted [here](https://www.google.com/maps/place/Albacete/@39.9945895,0.7358732,7.25z/data=!4m6!3m5!1s0xd667c1d58c47ad1:0x30340f3be4bc950!8m2!3d38.8036957!4d-1.9881527!16zL20vMDEyNDhs?entry=ttu).
>
>Conversely, Álava is located in the northern part of Spain, as shown [here](https://www.google.com/maps/place/%C3%81lava/@42.6107525,-1.6193495,9.75z/data=!4m6!3m5!1s0xd4fc01662943233:0x304f58273c9c940!8m2!3d42.9099989!4d-2.6983868!16zL20vMGh2azc?entry=ttu).
>
>Upon review, it's apparent that Albacete is incorrectly placed in two different locations in your database. The entry in the northern part of Spain should indeed be listed as Álava.
>
>For further reference, you can view and edit the data directly through this link: [Edit Link](https://writefield.nextzen.org/place/edit?url=https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-es/master/data/856/826/15/85682615.geojson).
>
>In summary, the issue highlights the misplacement of Albacete in two separate locations, with the correct designation being Álava for the northern region of Spain. Additional information and verification can be found [here](https://spelunker.whosonfirst.org/search/?q=albacete).

TLDR: [This record](https://spelunker.whosonfirst.org/id/85682615/) has the geometry of Álava, but a mix of names for both Álava and Albacete.

**Comment by stepps00** [comment]
Because this record has names for both Albacete (incorrect) and Álava (correct), I _think_ this record can be updated in place, rather than a new record being created.

If a new record is created, the change will need to include PIP work so descendant records know about the new region id.

Looking at all of the region records in Spain, this is the only issue with names - the correct Albacete record is ok as-is.

**Comment by nvkelso** [comment]
Related open data PR: https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/39

**Comment by nvkelso** [comment]
Agree that the existing WOF record is for the correct place, with a mix of true and confusing names, can be used for an in-place edit rather than triggering a new WOF ID.

However, the PR https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/39 need to be updated before being merged to cleanup all the `name:*` and `label:*` properties, not just the few otherwise we'll perpetuate the problem. That's easier to do outside of Write Field since there are so many of them for this place. 

Let's also review the concordances between the two records (especially Wikidata ID) to make sure we don't re-introduce name problems later.

**Comment by ghost** [comment]
Has there been any progress made on this?  We have a Pelias cluster ingested with WOF data and we just noticed this issue cropping up for our users in Spain.

**Comment by stepps00** [comment]
The record for Alava was updated in July, removing "Albacete" names: https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/42/files

I think the initial issue is actually resolved, but @alex-habrock-SHG are you still encountering issues with this record?

**Comment by ghost** [comment]
@stepps00 Thank you for your reply.  We are, but we also ingest bi-annually.  We should have this resolved in the next update.

Thank you again!
